gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-core/src/test/java/io/gravitee/am/gateway/handler/root/resources/endpoint/logout/LogoutEndpointHandlerTest.java

```diff
@@ -162,6 +162,33 @@ public void shouldInvokeLogoutEndpoint_targetUrl_client_restriction() throws Exc
                 HttpStatusCode.FOUND_302, "Found", null);
     }
 
+    @Test
```

Comment:
Some TestCases are missing.

* isRedirectUriWildcardAllowed and valid target_url provided with parameter

Reply:
more tests added